clipper_admin/clipper_admin/clipper_admin.py

```diff
@@ -117,6 +121,77 @@ def start_clipper(
             logger.warning("Error starting Clipper: {}".format(e.msg))
             raise e
 
+        def parse_configuration(config_path):
+            """ if a configuration file exists,
+            parse it and accordingly deploy models and applications
+            Parameters
+            ----------
+            config_path : str
+                Path to the configuration file.
+            """
+            if (config_path):
```

Comment:
Instead of checking for the existence of `config_path` here, check for it in the method that calls `parse_configuration` and only call `parse_configuration()` if `config_path` is not None.

Reply:
Done